examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -135,6 +133,21 @@ export class TaskList extends DataObject<{ InitialState: IBaseDocumentInitialSta
 		return this._draftData;
 	}
 
+	private get externalTaskListId(): string {
+		this._externalTaskListId = this.root.get("externalTaskListId");
```

Comment:
Let's extract `externalTaskListId` into a class-level constant so it doesn't get out of sync between set and get.